### DISCUSSION TITLE
Palworld RCON behaves a bit weird

### DISCUSSION THREAD

**Discussion by EpicLPer** [shupershuff/PalworldServerTools#7] [Q&A] [answered]
This is essentially a copy-paste from here, in case people in this repo know a bit more about this issue or have a potential solution for it: https://github.com/radj307/ARRCON/discussions/28

----

Heya,

I'm currently trying to get RCON working (to use this project here: https://github.com/shupershuff/PalworldServerTools ) tho I'm not sure what or why ARRCON is often times just giving an "empty string" as a response.

When typing the command `/ShowPlayers` in the Palworld textbox it does give me back the info properly:
![image](https://github.com/radj307/ARRCON/assets/8442196/ef42b322-877e-4276-b1d1-13d00d3ce80e)

However doing the same via RCON simply just throws an `[empty response]`:
![image](https://github.com/radj307/ARRCON/assets/8442196/57cda099-6165-4ff4-9ad7-d6b5fbed9385)

Same with any other commands I try that should give me some kind of response back.  
However, running commands like `broadcast Test123` does work just fine, however again with `[empty response]` in ARRCON.
![image](https://github.com/radj307/ARRCON/assets/8442196/555fb0eb-424a-4942-ad35-15057405b77e)

I'm not sure if Palworld or ARRCON is "at fault" here, or whatever else in between is doing something odd.

**Comment by shupershuff** [comment]
Hi there,
Thanks for the question. I'll certainly take a look and see what's going on to see where the issue lies, but know it may not be for a week or two as I'm quite busy with other things at the moment.
Hoping to give this script some love when I get some more time.
Stay tuned :)

**Reply by EpicLPer** [reply, marked as answer]
Heya, thanks for the reply! I think it's an ARRCON issue as another project called "rcon-cli" works just fine. I've reported this issue over at ARRCON now: https://github.com/radj307/ARRCON/issues/29
